changelog.md

```diff
@@ -1,7 +1,9 @@
 # MumieTask - Changelog
 
 All important changes to this plugin will be documented in this file.
-
+## [v2.1.0]
+### Changed 
+- Added gradepool implementation
```

Comment:
This is a new feature and should be in the section ###Added

The target audience for this change log are other ilias developers and maybe even some admins. They probably don't know what you mean with that description.

I suggest something like this;

`Sharing grades for the same MUMIE problems with other Ilias repositories can now be disabled.
`